src/components/Pill/PillButton/PillButton.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import { IoIosArrowUp, IoIosArrowDown } from 'react-icons/io';
+import { CROSS_CHAR, ENTER_KEY } from '../../../constants';
+import styles from './PillButton.scss';
+
+const { block, elem } = bem({
+    name: 'PillButton',
+    classnames: styles,
+    propsToMods: ['isActive', 'isOpen'],
+});
+
+const PillButton = React.forwardRef((props, ref) => {
+    const { isOpen, togglePopup, onClear, name, label, ...rest } = props;
+    const isActive = !!label;
+    const propsForBem = { ...props, isActive };
+
+    let buttonIcon;
+    let useButtonClick = false;
+
+    if (isOpen) {
+        buttonIcon = <IoIosArrowUp />;
+    } else if (isActive) {
+        buttonIcon = CROSS_CHAR;
+        useButtonClick = true;
+    } else {
+        buttonIcon = <IoIosArrowDown />;
+    }
+
+    const buttonClick = useButtonClick
+        ? e => {
+              e.stopPropagation();
+              onClear();
+          }
+        : undefined;
```

Comment:
let's try to follow `function` naming rule for functions inside hooks component rather than making it `const`. Since we're working with hooks it's nice to quickly distinguish functions from actual const variables just by having a quick look into the file.

Reply:
so far we mainly use arrow functions. if you want to change that standard, please bring it up as a proposal in a meeting or in the slack channel.